cypress/support/step-definitions/text-editor-steps.js

```diff
@@ -0,0 +1,59 @@
+import {
+  textEditorInput,
+  textEditorCounter,
+  textEditorToolbar,
+  innerText,
+  innerTextList,
+  innerTextLink,
+} from '../../locators/text-editor';
+import { positionOfElement } from '../helper';
+
+When('I type {string} in Text Editor', (text) => {
+  textEditorInput().clear().type(text);
+});
+
+When('I focus the Text Editor', () => {
+  textEditorInput().focus();
+});
+
+Then('Text Editor counter shows {string} characters left', (charactersLeft) => {
+  textEditorCounter().should('have.text', charactersLeft);
+});
+
+When('I click onto {string} in Text Editor Toolbar', (buttonType) => {
+  textEditorToolbar(buttonType).click();
+});
+
+Then('text has {string} css property', (buttonType) => {
+  if (buttonType === 'bold') {
+    innerText().should('have.css', 'font-weight', '700');
+  } else if (buttonType === 'italic') {
+    innerText().should('have.css', 'font-style', 'italic');
+  }
+});
+
+Then('text is rendered in {string} type', (buttonType) => {
+  if (buttonType === 'bullet-list') {
+    innerTextList('ul', positionOfElement('second')).should('have.text', 'Testing');
+    innerTextList('ul', positionOfElement('third')).should('have.text', 'is');
+    innerTextList('ul', positionOfElement('fourth')).should('have.text', 'awesome');
+  } else if (buttonType === 'bullet-list-numbers') {
+    innerTextList('ol', positionOfElement('second')).should('have.text', 'Testing');
+    innerTextList('ol', positionOfElement('third')).should('have.text', 'is');
+    innerTextList('ol', positionOfElement('fourth')).should('have.text', 'awesome');
+  }
+});
+
+Then('button {string} is clicked and active', (buttonType) => {
```

Comment:
why this step is `clicked and active` if you are checking only one assertion?
```suggestion
Then('button {string} is clicked', (buttonType) => {
```

Reply:
it means that this button was clicked and active means it wasn't `unclicked` / `disactivated`